core/domain/stats_jobs_continuous.py

```diff
@@ -623,6 +628,8 @@ def reduce(key, stringified_values):
                         'since the latest exploration version is %s' % (
                             latest_version, exp.version))
                     versions = []
+            elif not exp:
```

Comment:
This is no longer necessary due to the check in line 612, right?

Reply:
Ah yes, correct. Removed.